src/components/ChartScale/ChartScale.css

```diff
@@ -27,13 +27,15 @@
 @media only screen and (min-width: 992px) {
```

Comment:
I am still seeing the axis change on mobile size, maybe we should disable that for now, it looks wacky without the right placement.